official/utils/logs/logger.py

```diff
@@ -36,13 +36,17 @@
 import tensorflow as tf
 from tensorflow.python.client import device_lib
 
+from official.utils.logs import cloud_lib
```

Comment:
Note to selves: BUILD rules required for this and the test above.

Reply:
ACK. Will do